scripts/benchmarks/data_processing.py

```diff
@@ -64,6 +64,7 @@ def _build_data_formatting_func(
             "response_field must be specified if tokenize=True and response_template=None."
 
     def _format(example):
+        nonlocal loss_masking # reference to variable in _build_data_formatting_func
```

Comment:
put a comment to say that format is going to be passed into `map` , wherby if `loss_masking` is not rebound using `nonlocal`, then the spawned processes will have no reference to it.